{"name": "Unsuccessfully login with email: simeon.hhl.qa@gmail.co and password: Test#1234", "status": "passed", "steps": [{"name": "Check for error", "status": "passed", "start": 1732118942720, "stop": 1732118944387}, {"name": "Check the error text", "status": "passed", "start": 1732118944387, "stop": 1732118944404}], "parameters": [{"name": "email", "value": "'simeon.hhl.qa@gmail.co'"}, {"name": "password", "value": "'Test#1234'"}, {"name": "error", "value": "'There was a problem logging you in. Please try again.'"}, {"name": "driver", "value": "'edge'"}], "start": 1732118941736, "stop": 1732118944404, "uuid": "8c78fa22-e5c3-4e51-a9ed-b5c33d761d72", "historyId": "2ffef19ed6000db1495424120c5ac1a1", "testCaseId": "c2811b1721d8231bbb0fb40778ae099c", "fullName": "Tests.test_Login.TestLogin#test_2", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "tag", "value": "flaky(reruns=3, reruns_delay=1)"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az1251-146"}, {"name": "thread", "value": "1799-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}